pallet/src/signer.rs

```diff
@@ -80,8 +79,11 @@ where
 {
     /// The signers of a script transaction.
     signers: BoundedBTreeMap<AccountId, SignerData<Balance>, Size>,
-    /// The block number when this `Multisig` was created and stored.
-    block_height: BlockNumber,
+
+    /// The block height at which this `Multisig` was initally stored.
+    ///
+    /// Used only for multisigure purposes.
+    stored_block_height: Option<BlockNumber>,
```

Comment:
For what will it now be used too? Why do we need it optional?

Reply:
So when we create ScriptSignatureHandler for single signer scripts, we don't need this block information since we won't be able to store it in the Storage. So, for that case, we shall have `None` here

Reply:
Done